website/docs/r/group_profile.html.markdown

```diff
@@ -0,0 +1,47 @@
+---
+layout: "turbot"
+title: turbot
+template: Documentation
+page_title: "Turbot: turbot__group_profile"
+nav:
+  title: turbot__group_profile
```

Comment:
This title is wrong?

Reply:
These terraform internal strings like turbot__group_profile are not helpful for users in any scenario. Suggest we stop using them in any docs / release notes (unless it's really a TF standard doc format).